fotoapparat/src/main/java/io/fotoapparat/capability/provide/CapabilitiesProvider.kt

```diff
@@ -21,7 +21,7 @@ private fun SupportedParameters.getCapabilities(): Capabilities {
             canSmoothZoom = supportedSmoothZoom,
             maxMeteringAreas = maxNumMeteringAreas,
             jpegQualityRange = jpegQualityRange,
-            antiBandingModes = supportedAutoBandingModes.extract { it.toAntiBandingMode() },
+            antiBandingModes = supportedAutoBandingModes.extract(String::toAntiBandingMode),
```

Comment:
I am a bit sceptical about the `::` pattern (there are few in other files too) but I don't have hard feelings